WikiWatershed/model-my-watershed
# Unit Conversion: TR-55

## Overview

Implements unit conversion for TR-55.

Connects #3023 

### Demo

![image](https://user-images.githubusercontent.com/1430060/48650768-960c0780-e9c5-11e8-9e8b-cdc2350a27b7.png)

![image](https://user-images.githubusercontent.com/1430060/48650780-9d331580-e9c5-11e8-8416-7de1d73bd9d8.png)

![image](https://user-images.githubusercontent.com/1430060/48650801-b3d96c80-e9c5-11e8-94f1-61f634252de6.png)

![image](https://user-images.githubusercontent.com/1430060/48650808-ba67e400-e9c5-11e8-8063-d0752834b2c1.png)

## Testing Instructions

* Check out this branch and `bundle`
* Make sure you are logged in. In your User Profile, set unit scheme to US Customary
* Make a TR-55 project
* Ensure the Current Conditions model results are shown in US Customary units
* Ensure the precipitation slider is in inches, and that changing it updates the results correctly
* Add a new scenario. Draw a shape. Ensure its area is displayed in miles.
* Open Compare view. Ensure all the charts and tables there are in US Customary units.
* Download a CSV of the Compare View from the top right button. Ensure it is in US Customary units.
* Change your preference to Metric and test again.
